packages/toolpad-core/src/AppProvider/AppProvider.tsx

```diff
@@ -126,6 +126,11 @@ export interface AppProviderProps {
    * @default window
    */
   window?: Window;
+  /**
+   * The nonce to be used for inline scripts.
+   * @default undefined
```

Comment:
I think it shouldn't need this @default when it's undefined.

Reply:
I was in the process of generating the docs while you were reviewing.  Updated and pushed

You are correct about @default not being necessary when the default is undefined